app/jobs/message_broadcast_job.rb

```diff
@@ -1,10 +1,10 @@
-class NewMessageBroadcastJob < ApplicationJob
+class MessageBroadcastJob < ApplicationJob
   queue_as :default
 
-  def perform(message)
+  def perform(message, dom_id)
```

Comment:
Maybe since this method 1) receives the message object, 2) is already responsible for converting it to HTML, and 3) is also responsible for the `ActionCable.server.broadcast` call, it makes sense for this method to also be responsible for calculating the `message_dom_id`.

This would make the call sites a little simpler (and in general prevent changes to the method signature of `perform` here), which seems like a good win for testability.